### DISCUSSION TITLE
HTML reporting like htmlextra, allure or others

### DISCUSSION THREAD

**Discussion by Benjamiche** [usebruno/bruno#1550] [Q&A]
Hello,

I'm new to Bruno and starting to migrate API tests from Postman. With Postman (cli execution), I use newman to execute and generate html reports like htmlextra or allure-report.

When I use Bruno's "--output results.xml format junit" command, a lot of information (headers, body, response, ...) is not used and therefore not displayed.
The allure report generated is too small without details.

Is it possible to extend junit.js here [https://github.com/usebruno/bruno/blob/main/packages/bruno-cli/src/reporters/junit.js] or is there another way?

Thank you

**Comment by amaillet2024** [comment]
I think you should have a look to this discussion : 
https://github.com/usebruno/bruno/issues/1659
